### PR TITLE
add ion-tree-list w/ git auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
Pull request for issue: #9340 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.
@x09326 @pvnr0082t Please help me review this PR, thanks!
Manually add some of old versions from git repo because the naming scheme is different from the latest version.
Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/fer/ion-tree-list
 * Official website (optional, not the repository): https://fer.github.io/ion-tree-list/
 * NPM package url (optional):
 * License and its reference: MIT, https://github.com/fer/ion-tree-list/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 47
   - Count of watchers: 7
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/ion-tree-list/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "ionic",
+    "ion",
+    "tree",
+    "list"
+  ],
+  "author": "fer <fer@ferqwerty.com>",
+  "license": "MIT",
+  "homepage": "https://fer.github.io/ion-tree-list/",
```
Comment (x09326) [inline ajax/libs/ion-tree-list/package.json]: @kennynaoh I think this is a demo page, we don't need it.

Comment (kennynaoh) [inline ajax/libs/ion-tree-list/package.json]: I think there is a icon which can link to git repo, so I add this as homepage.
@PeterDaveHello Could you help me determine whether this website can be added? Thanks!

Comment (PeterDaveHello) [inline ajax/libs/ion-tree-list/package.json]: Fine to use this.

Comment (kennynaoh) [inline ajax/libs/ion-tree-list/package.json]: @x09326 Please help me review this PR again, thanks!

**Comment by x09326** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
What's the naming difference?? I didn't see a need of manually adding old versions.

**Comment by kennynaoh** [comment]
The `ion-tree-list.css` file is minified file in those old versions, and the author offer the `ion-tree-list.min.css` for minified css file. So I change the `ion-tree-list.css` to `ion-tree-list.min.css`.

**Pull request merged**